src/monet.js

```diff
@@ -177,6 +177,22 @@
         return listFindC(l, fn).run()
     }
 
+    function listContainsC(l, val) {
+        if (l.isNil) {
+            return Return(false)
+        }
+        var h = l.head()
+        return h === val ?
```

Comment:
Look that if we have `a = ""` and `b = ""` it'll be returned with `true` from first condition.

This line is an optimisation to avoid `function` checks. 

If you don't like it it can be dropped, but it's not that evil.